### PR TITLE
Add new keymap with split shift and split backspace for bananasplit PCB

### PR DESCRIPTION
**Pull request by christopherjanzen**
## Description

Provides an HHKB-like layout for the Bananasplit PCB

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/thevankeyboards/bananasplit/keymaps/cijanzen/config.h [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2017 Balz Guenat
```
Comment (noroadsleft) [inline keyboards/thevankeyboards/bananasplit/keymaps/cijanzen/config.h]: This file can be deleted.

Comment (christopherjanzen) [reply keyboards/thevankeyboards/bananasplit/keymaps/cijanzen/config.h]: I've removed the file now

File: keyboards/thevankeyboards/bananasplit/keymaps/cijanzen/keymap.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    KC_LSFT,      KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,  KC_DOT,  KC_SLSH,   KC_RSFT, MO(1), \
+    KC_LCTL,    KC_LGUI,    KC_LALT,    KC_SPC,    KC_SPC,     KC_SPC,    KC_RALT,  KC_RGUI,   _______, KC_MENU, KC_RCTL
+  ),
+
+  [HHKB_NAV_LAYER] = LAYOUT_hhkb_arrow( \
+    RESET, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_INS, KC_DEL, \
+    _______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_PSCR,   KC_SLCK,   KC_PAUS,   KC_UP, _______, KC_DEL, \
+    _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_HOME,   KC_PGUP, KC_LEFT,  KC_RGHT, KC_ENT, \
+    _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_END,  KC_PGDN,  KC_DOWN,  _______, _______, \
+    _______,  _______,  _______, _______,      _______,      _______,  _______, _______,  _______, _______, _______
```
Comment (fauxpark) [inline keyboards/thevankeyboards/bananasplit/keymaps/cijanzen/keymap.c]:
```suggestion
  [DEFAULT_LAYER] = LAYOUT_hhkb_arrow(
    KC_ESC,  KC_1,  KC_2,  KC_3,  KC_4,  KC_5,  KC_6,  KC_7,  KC_8,  KC_9,  KC_0,  KC_MINS,  KC_EQL,  KC_BSLS, KC_GRV,
    KC_TAB,  KC_Q,   KC_W,   KC_E,   KC_R,   KC_T,   KC_Y,   KC_U,   KC_I,   KC_O,   KC_P,   KC_LBRC, KC_RBRC, KC_BSPC,
    KC_CAPS,   KC_A,   KC_S,   KC_D,   KC_F,   KC_G,   KC_H,   KC_J,   KC_K,   KC_L,   KC_SCLN,  KC_QUOT,       KC_ENT,
    KC_LSFT,      KC_Z,   KC_X,   KC_C,   KC_V,   KC_B,   KC_N,   KC_M,   KC_COMM,  KC_DOT,  KC_SLSH,   KC_RSFT, MO(1),
    KC_LCTL,    KC_LGUI,    KC_LALT,    KC_SPC,    KC_SPC,     KC_SPC,    KC_RALT,  KC_RGUI,   _______, KC_MENU, KC_RCTL
  ),

  [HHKB_NAV_LAYER] = LAYOUT_hhkb_arrow(
    RESET, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_INS, KC_DEL,
    _______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_PSCR,   KC_SLCK,   KC_PAUS,   KC_UP, _______, KC_DEL,
    _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_HOME,   KC_PGUP, KC_LEFT,  KC_RGHT, KC_ENT,
    _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   KC_END,  KC_PGDN,  KC_DOWN,  _______, _______,
    _______,  _______,  _______, _______,      _______,      _______,  _______, _______,  _______, _______, _______
```

**Comment by fauxpark** [comment]
Thanks!

**Pull request merged**